verify.py

```diff
@@ -108,10 +118,20 @@ def __init__(self, configfile, auth, logger):
             elif key == "rdfLang":
                 self.lang = value
 
-        # if not specified in config, set default ext & lang to turtle
+        # if lang not specified in config, set the ext & lang to turtle
         if not hasattr(self, 'lang'):
             self.ext = '.ttl'
             self.lang = 'text/turtle'
+        # set the ext based on the rdfLang
+        else:
+            if self.lang in EXT_MAP:
+                self.ext = EXT_MAP[self.lang]
+            else:
+                logger.error(
+                    'Unrecognized rdf dialect specified in config file!'
+                    )
+                print('Unrecognized rdf dialect specified in config file!')
```

Comment:
What would be the protocol here -- I'm assuming I make the edits? Can I do that directly in Github?